manual_constraints.txt

```diff
@@ -1,3 +1,4 @@
 sphinx >= 4.5, <6.2.0
 sphinx_autodoc_annotation >= 1.0
-sphinx_book_theme ~= 1.0.1
\ No newline at end of file
+sphinx_book_theme ~= 1.0.1
+qiskit-algorithms ~= 0.2.1
```

Comment:
I think we should omit this constraint; we can add it if we encounter a problem with 0.3...

Reply:
Fixed a mistake in the previous commit